packages/venia-concept/src/components/CreateAccount/createAccount.js

```diff
@@ -20,7 +20,16 @@ class CreateAccount extends Component {
             createAccountError: PropTypes.string
         }),
         createAccountError: PropTypes.object,
-        createAccount: PropTypes.func
+        createAccount: PropTypes.func,
```

Comment:
It looks like this component now expects to receive `props.onSubmit` rather than `props.createAccount`, so this prop type needs to be updated.

Reply:
Yeah, definitely, I will change this.